### PR TITLE
added: extra product info in missing sku modal

### LINKED ISSUE #198: Rich product info for missing products
## What is the motivation for adding/enhancing this feature?

The missing products modal should show rich product info after a user has successfully mapped the unidentified product ID with a product in the system.

### PR DESCRIPTION
**Pull request by disha1202**
Additional Changes:
fix: product should update on enter
fix: sku not updating in completed tab(#85zrpv6dd)

### Related Issues

Closes #198  #199 

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2023-03-06 11-42-21

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/import#contribution-guideline)

### CODE REVIEW

File: src/components/MissingSkuModal.vue [hunk 1]
```diff
@@ -41,8 +41,12 @@
         </ion-list>
 
         <ion-list v-if="segmentSelected === 'completed'">
-          <ion-item v-for="item in getCompletedItems()" :key="item.shopifyProductSKU">
+          <ion-item v-for="item in getCompletedItems()" :key="item.shopifyProductSKU" lines="full">
```
Comment (dt2patel) [inline src/components/MissingSkuModal.vue:44]: why have we added lines="full"?

File: src/components/MissingSkuModal.vue [hunk 2]
```diff
@@ -41,8 +41,12 @@
...
+            <ion-thumbnail slot="start">
+              <Image :src="item.imageUrl" />
+            </ion-thumbnail>
             <ion-label>
+              <p>{{ item.parentProductName }}</p>
```
Comment (dt2patel) [inline src/components/MissingSkuModal.vue:49]: Use class="overline" whenever placing content about the primary piece of info in an ion-item

**Review by dt2patel — COMMENTED** [review]
Added my feedback, please check with someone else for functional feedback

File: src/components/MissingSkuModal.vue (hunk 1 above)
Comment (disha1202) [reply src/components/MissingSkuModal.vue:44]: Looked better with lines="full" 
as compared to this

Screenshot from 2023-03-06 12-23-46

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**